stories/.eslintrc

```diff
@@ -3,6 +3,15 @@
     "import/no-extraneous-dependencies": ["error", {
       // Allow dev-dependencies in this directory.
       "devDependencies": true
-    }]
+    }],
+    "import/no-unresolved": [
+      "error", {
+        // TODO: Look for a better way.
+        // This was the only way I could disable the
+        // tests resolve linting issue for now (without
+        // adding disable eslint [this rule] to the file itself).
+        "ignore": [ "tests" ]
```

Comment:
I suppose that's fine for now. We should probably use the [`webpack`](https://www.npmjs.com/package/eslint-import-resolver-webpack) resolver but I couldn't get it to work.